### PR TITLE
Created single video page

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made
- Created a single video page
- Added static icons for features: like, add to playlist, watch later

### Preview
[Link](https://deploy-preview-14--always-stream.netlify.app/)

### CODE REVIEW

File: src/components/VideoCard/VideoCard.css [hunk 1]
```diff
@@ -21,7 +21,7 @@
   text-overflow: ellipsis;
 }
 
-.avatar-sm {
+.avatar-video-card {
   width: 2.5rem;
```
Comment (amartya-rs) [inline src/components/VideoCard/VideoCard.css:25]: suggestion: can use aspect-ratio: 1/1

**Review by amartya-rs — APPROVED** [review]
Hey Sanket, 
Your code looks fine to me. 
And great job on the UI.

File: src/components/VideoCard/VideoCard.css (hunk 1 above)
Comment (SanketDhabarde) [reply src/components/VideoCard/VideoCard.css:25]: Ok, I will look into it.

**Review by jeetbhalani15 — APPROVED** [review]
Single video page works fine
- Code looks readable
- Nice UI

**Pull request merged**